src/basic/FStar.Options.fs

```diff
@@ -778,6 +782,47 @@ let rec specs_with_types () : list<(char * string * opt_type * string)> =
         Const (Bool true),
         "Parses and prettyprints in place the files included on the command line");
 
+       ( 'f',
+        "force",
+        Const (Bool true),
+        "Force checking the current file even if a checked file exists for it");
+
+       ( noshort,
+        "fuel",
+        PostProcessed
+            ((function | String s ->
+                         let p f = Int (int_of_string f) in
+                         let min, max =
+                           match split s "," with
+                           | [f] -> f, f
+                           | [f1;f2] -> f1, f2
+                           | _ -> failwith "unexpected value for --fuel"
```

Comment:
Can we change such `failwith`(s), that are not "Impossible!", to `raise_error` with `Errors.FailToProcessPragma`?

Reply:
These are actually caught by our getopt and we get an error like this:
```
$ ./bin/fstar.exe --fuel 1,1,1 ulib/prims.fst 
wrong argument given to option `fuel`
```
So I guess raising anything is equivalent... I could rework this getopt logic I guess (and maybe use something more standard?)

Reply:
This is fine too, as long as in the emacs mode the cursor does not go to the beginning of the file and says "F* may be in an inconsistent state" (which is what happens with `failwith` normally I think?).